From a0de125c5afb181eac04d93a21a3da62776dad6c Mon Sep 17 00:00:00 2001
From: Jarno Rajahalme <jarno@isovalent.com>
Date: Sat, 8 Jan 2022 11:06:29 +0200
Subject: [PATCH] envoy: Note unreferenced parameter

Fix build error:

external/envoy/source/server/active_tcp_listener.cc:45:40: error: unused variable 'chain' [-Werror=unused-variable]

Signed-off-by: Jarno Rajahalme <jarno@isovalent.com>
---
 source/server/active_tcp_listener.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/source/server/active_tcp_listener.cc b/source/server/active_tcp_listener.cc
index 3e0f850c9..2f4c07536 100644
--- a/source/server/active_tcp_listener.cc
+++ b/source/server/active_tcp_listener.cc
@@ -43,6 +43,7 @@ ActiveTcpListener::~ActiveTcpListener() {
   }
 
   for (auto& [chain, active_connections] : connections_by_context_) {
+    UNREFERENCED_PARAMETER(chain);
     ASSERT(active_connections != nullptr);
     auto& connections = active_connections->connections_;
     while (!connections.empty()) {
-- 
2.34.0

